spec/javascripts/school_administrator_dashboard/StudentsTable.test.js

```diff
@@ -21,7 +21,7 @@ describe('Dashboard Students Table', () => {
     expect(cellTexts).toEqual([
       "Pierrot Zanni",
       "3",
-      moment.utc().subtract(3, 'months').format('M/D/YY'),
+      moment().subtract(3, 'months').format('M/D/YY'),
```

Comment:
@edavidsonsawyer Can you say more about this, shouldn't this change fail the test?

Reply:
You know, at the time I didn't think so because I assumed the SST data needed to do something like moment(threemonthsago).utc instead of moment.utc(threemonthsago) but going over the moment docs again I guess that's wrong. So it should fail the test.

Here's the thing. The test only stopped failing after I removed utc here. If you check out the travis error on my initial commit you'll see it was getting inconsistent dates between the spec and the test data. I'll work through this a bit more later tonight and see if I can figure out what's up here. Out of curiosity, why do you parse the SST date as utc?

Reply:
@kevinrobinson actually, thinking about this again, I see what's up. The threeMonthsAgo variable creates a local moment and subtracts three months. The SST date parses that string as utc with moment.utc(threeMonthsAgo), but that doesn't change the time, it just sets the utc offset to 0.  

In the spec, moment.utc().subtract(3, 'months) appropriately gives you a date 3 months from the current UTC time since it creates a utc moment and subtracts three months from that. So that won't be the same as the date above and this spec will fail for any code I push after dinner.

Reply:
I don't know how important it is to keep the SST date in utc. We could fix it by creating the initial variable in utc. We could use moment(threeMonthsAgo).utc() to get the SST test date and revert this change we're commenting on. It kind of looks like we could also just hard code the date since I don't believe we're testing any code that requires the SST date to happen within a specific timeframe, but I could be wrong here.

Reply:
Nevermind all that. That odd parsing behavior seems like an artifact of the way those dates were formatted. If they're changed to ISO 8601 then utc parsing works like you'd expect, so I just did that.

Reply:
@edavidsonsawyer Yeah this is great to bring up.  In general, my default assumption has been "store all times in UTC always," and then if we want to project them to local time, only do that at the very end when we show it to the user.  But we probably don't handle that entirely correctly in a bunch of places.  The new home page will surface this in https://github.com/studentinsights/studentinsights/pull/1505/files#diff-8194c0c736a7a4045e1d0a21d7715997R46, so we can sort it out there.

Separately, in code and in tests I've found it can be helpful to make the "now" time explicit, so that in tests all runs are deterministic and consistent.  The way you tackled that with making all the times relative is great too, but sometimes it's clearer in the test to write concretely "the date is 3/5/18" and so events that happened on "12/19/17" shouldn't show up.